service/src/main/java/bio/terra/cbas/common/exceptions/WorkflowAttributesNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.cbas.common.exceptions;
+
+public class WorkflowAttributesNotFoundException extends Exception {
+
+  public WorkflowAttributesNotFoundException(String message) {
+    super(message);
+  }
```

Comment:
Today this message is always the same, and kind of just duplicates the name of the exception. Maybe we could template the message inside the class, and the caller could give us some useful debugging information on what was missing? eg something like
 ```suggestion
   public WorkflowAttributesNotFoundException(String attribute, String recordId, String inputName) {
     super("Attribute %s not found in WDS record %s (to populate workflow input %s).".formatted(attribute, recordId, inputName));
   }
```